Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.13
+
+WORKDIR .
+
+COPY /app/main.py .
+
+RUN apt-get update
+RUN apt-get install -y python3-pip
+RUN pip install "fastapi[standard]"
+
+COPY . .
```

Comment:
do not copy the whole projects, copy only files you need to run an app. additionally, move this copy to the top